internals/getCsp.js

```diff
@@ -18,23 +21,25 @@ const csp = ({ extension } = {}) =>
       'report-sample';
     style-src
       'self'
-      fonts.googleapis.com
+      https://fonts.googleapis.com
       'unsafe-inline'
       'report-sample';
     font-src
       'self'
-      fonts.gstatic.com;
+      https://fonts.gstatic.com;
     connect-src
       'self'
-      grpc.oasis.dev
-      testnet.grpc.oasis.dev
-      api.oasisscan.com
-      monitor.oasis.dev;
+      https://grpc.oasis.dev
+      https://testnet.grpc.oasis.dev
+      https://api.oasisscan.com
+      https://monitor.oasis.dev
+      ${extension ? extensionWebsocket : ''}
+      ;
     img-src 'self' data: https:;
```

Comment:
> resolve issue with static assets. Mainly usage of <Avatar /> which requires a string url prop. Stuff like https://parceljs.org/recipes/image/#javascript won't work in extension. We may add a copy task before starting a ext build, or just add support for inline svgs and get rid of Avatar usage in logotype in app header, or it may be handled by manifest somehow

Oof
related docs https://parceljs.org/recipes/web-extension/#asset-urls

But this somehow works (related docs https://parceljs.org/getting-started/migration/#importing-non-code-assets-from-javascript)
```tsx
import logo from '../../../logo192.png'
          <Avatar src={logo} size={sizeLogo[size]} />
```

and also (related docs https://parceljs.org/features/bundle-inlining/)
```tsx
import logo from 'data-url:../../../logo192.png'
          <Avatar src={logo} size={sizeLogo[size]} />
```

Reply:
cool thanks, I will go with not inline option as this will require csp updates